File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.py
def solution(my_string):
    a = ''
    for i in my_string:
        if i in ['1','2','3','4','5','6','7','8','9','0']:
            a = a+i
        else:
            a = a+ ' '
    l = list(map(int,a.split()))
    return sum(l)

File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        StringBuilder sb = new StringBuilder();
        for (char c : my_string.toCharArray()) {
            if (Character.isDigit(c)) {
                sb.append(c);
            } else {sb.append(" ");}
        }
        String s = sb.toString();
        String[] nums = s.split("\\s+");
        for (String c : nums) {
            if (!c.isEmpty()) {
                answer += Integer.parseInt(c);
        
            }
        }
        return answer;
    }
}